### PR TITLE
Clarify 5b5f

### PR DESCRIPTION
**Pull request by Nanush7**

### CODE REVIEW

File: wca-guidelines.md [hunk 1]
```diff
@@ -111,6 +111,7 @@ To be more informative, each Guideline is classified using one of the following
 - 5b5f+) [EXAMPLE] There is a [Visual Guide](https://drive.google.com/file/d/15XszaCGNvy3Dk6X6qERzZWZaDH1RH04z/view) available for [Regulation 5b5f](regulations:regulation:5b5f).
 - 5b5f++) [EXAMPLE] A 4x4x4 Cube edge "wing" twisted on its spot is considered solved.
 - 5b5f+++) [EXAMPLE] A corner affected by a corner twist is considered to be placed in a valid position, but such position does not render the piece solved.
+- 5b5f++++) [CLARIFICATION] If there are multiple pieces partially detached or not fully placed, their final position is assesed one at a time. The piece that is closest to a valid spot is considered to take such spot, so that it is not available to other pieces.
```
Comment (CarterKoala) [inline wca-guidelines.md]:
```suggestion
- 5b5f++++) [CLARIFICATION] If there are multiple pieces partially detached or not fully placed, their final position is assessed one at a time. The piece that is closest to an open spot is considered first, followed by the next closest to an open spot. Once a spot is considered to be occupied by a piece, no other pieces may take that spot.
```
This is very tricky to explain. I like my suggestion a little bit more, but I think it's more confusing than anything

Comment (ohexter) [reply wca-guidelines.md]:
```suggestion
- 5b5f++++) [CLARIFICATION] If there are multiple pieces partially detached or not fully placed, their final position is assessed one at a time. The piece that is closest to an open position is assessed first, and is then considered to occupy that position. This process is repeated until all partially detached or not fully placed pieces have been assessed.
```
Another suggestion, this actually has a slightly different meaning from Carter's suggestion. I'd also like to see spot/place/position used consistently (and I'm not a fan of spot).

Comment (CarterKoala) [reply wca-guidelines.md]: I like Hex's suggestion :) @Nanush7 feel free to Commit if you agree

**Comment by Jobarion** [comment]
I'm posting this picture again because it's useful to illustrate some examples. P1 and P2 are partially attached pieces. S1 and S2 are free slots to occupy in the puzzle's mechanism.

This change say that:

1. P2 is assessed first because the distance between P2 and S1 is the minimal distance out of all combinations of pieces and slots.
2. P2 is placed into the slot S1 because that is the one that is closest to it.
3. P1 is placed into the last remaining free slot S2.

One note: We can invert the logic and start with the _slot_ that is closest to a piece. This would lead to S2 "claiming" P2, and S1 claiming P1, the reverse of what happened here. 

img_20240402_143920

**Comment by CarterKoala** [comment]
Wouldn't inverting the logic still have S1 taking P2? Either way, I think the way it is worded is what we want

**Pull request merged**